tests/automember/test_automember_client_context.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Test automember
+  hosts: ipaclients, ipaserver
+  become: no
+  gather_facts: no
+
+  tasks:
+  - name: Include FreeIPA facts.
+    include_tasks: ../env_freeipa_facts.yml
+
+  # Test will only be executed if host is not a server.
+  - name: Execute with server context in the client.
+    ipaautomember:
+      ipaadmin_password: SomeADMINpassword
+      ipaapi_context: server
+      name: ThisShouldNotWork
+      state: rebuild
```

Comment:
This made me look into the docs to see if this is supported. It is supported by the module, but there are no docs #648

Reply:
Thank you.